client.go

```diff
@@ -175,6 +175,8 @@ type marathonClient struct {
 	listeners map[EventsChannel]int
 	// a custom logger for debug log messages
 	debugLog *log.Logger
+	// wait time between repetitive requests to API
+	waitTime time.Duration
```

Comment:
Maybe be a bit more specific and call it `pollingWaitTime`. Would then also update the description to

`//  wait time between repetitive requests to the API during polling`

Likewise in the `Config` definition.